src/modules/internet/index.ts

```diff
@@ -455,4 +462,27 @@ export class Internet {
       this.faker.definitions.internet.emoji[emojiType]
     );
   }
+
+  /**
+   * Generates a random status code.
+   *
+   * @param options Options object.
+   * @param options.types A list of the HTTP status code types that should be used.
+   * @example
```

Comment:
```suggestion
   *
   * @example
```

Looks like we missed this one in emoji.